src/modules/Song/Song.js

```diff
@@ -12,32 +12,38 @@ function Song(props) {
   const [songListData, setSongListData] = useState({ actualSongListData: [], isSongListDataRetrieved: false });
 
 
-  // For Song Details:
-  // Pass in track ID -> tracks API -> (song) name, artistName
-  // From tracks API -> albumID for albums API
+  /* 
+    For Song Details:
+    Pass in track ID -> tracks API -> (song) name, artistName
+    From tracks API -> albumID for albums API
 
-  // From albums API -> released, label
-  // From albums API -> links { image { href = images API }} for album image API
+    From albums API -> released, label
+    From albums API -> links { image { href = images API }} for album image API
 
-  // From albums images API -> 170 x 170 pixels image
+    From albums images API -> 170 x 170 pixels image
+  */
 
 
-  // For Song Album:
-  // From albums API, use the API link to find tracks -> list of tracks each has: (song) name, previewURL
+  /*
```

Comment:
My multiline comments are like this:

```
  /*
  * Explanation of below code:
  * If data has been retrieved, check if there is any data to be displayed.
  * If there is, display the tracks. If not, display 'no songs found'.
  * If data hasn't been retrieved, display 'Loading...'
  */
```

What style should we follow?

Reply:
Idk how to write the comment like what you did above without manually putting the asterisks. Unless there is a shortcut or something that allows us to write the comment like above, I think we should do it my way.